applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -41,24 +41,36 @@ config DESKTOP_DVFS_RETRY_COUNT
 	  Number of retries of DVFS frequency change after which DVFS module will report
 	  MODULE_STATE_ERROR.
 
-config DESKTOP_DVFS_SMP_TRANSFER_TIMEOUT_MS
-	int "SMP transfer timeout"
-	default 5000
-	range 1 65535
-	help
-	  Timeout in milliseconds specifying time after which DVFS module will go back to
-	  low frequency after receiving SMP transfer event.
-
-config DESKTOP_DVFS_CONFIG_CHANNEL_TIMEOUT_MS
-	int "Config channel timeout"
-	default 5000
-	range 1 65535
-	help
-	  Timeout in milliseconds specifying time after which DVFS module will go back to
-	  low frequency after reciving config event.
-
 module = DESKTOP_DVFS
 module-str = DVFS
 source "subsys/logging/Kconfig.template.log_config"
 
+# Cannot leave this empty, as it will cause Kconfiglib to fail so add always met dependency
+dvfs_dependencies = DESKTOP_DVFS
+dvfs_state = DVFS_STATE_INITIALIZING
+# We start with DVFS_STATE_INITIALIZING on as it is active on start.
+config DVFS_STATE_INITIALIZING_INITIAL_ACTIVE
+	default y
+rsource "Kconfig.dvfs_state_template"
+
+
+dvfs_dependencies = CAF_BLE_COMMON_EVENTS
```

Comment:
`CAF_BLE_COMMON_EVENTS && (BT_MAX_CONN = 1)` (the dependency needs to also be removed from `menuconfig DESKTOP_DVFS`